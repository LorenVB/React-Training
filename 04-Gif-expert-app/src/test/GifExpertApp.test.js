const { shallow } = require("enzyme")
const { GifExpertApp } = require("../GifExpertApp")

describe('Testing GifExpertApp component', () => {
      const wrapper= shallow(<GifExpertApp />)
    it('should match with the snapshot', () => {
       
        expect(wrapper).toMatchSnapshot();
    })

    it('should match with the initial state', ()=>{
        expect(wrapper.find('GifGrid').prop('category')).toBe('One Punch');
    })
})
