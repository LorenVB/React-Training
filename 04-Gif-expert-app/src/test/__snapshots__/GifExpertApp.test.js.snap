// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing GifExpertApp component should match with the snapshot 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <GifGrid
    category="One Punch"
    key="One Punch"
  />
</Fragment>
`;
